flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -515,28 +514,23 @@ private static void start(YarnConfiguration conf, String principal, String keyta
 
 			File flinkConfDirPath = findFile(flinkDistRootDir, new ContainsName(new String[]{"flink-conf.yaml"}));
 			Assert.assertNotNull(flinkConfDirPath);
+			org.apache.flink.configuration.Configuration flinkCfg =
+					new org.apache.flink.configuration.Configuration();
```

Comment:
I think we should read in the flink configuration stored under `flinkConfDirPath` instead of creating an empty one.

Reply:
done